### PR TITLE
feat(client): add table and column name length validation

### PR DESCRIPTION
**Pull request by puzpuzpuz**

### CODE REVIEW

File: sender.go [hunk 1]
```diff
@@ -48,6 +48,8 @@ import (
 // chars found in table or column name.
 var ErrInvalidMsg = errors.New("invalid message")
 
+const fsFileNameLimit = 127
```
Comment (jerrinot) [inline sender.go:51]: this limit is configurable on server-side isn't it?

Comment (bluestreak01) [reply sender.go:51]: it is, this is also configurable limit on the server. Can this perhaps be made configurable on client too ?

**Review by bluestreak01 — CHANGES_REQUESTED** [review]

**Comment by bluestreak01** [comment]
could we have CI setup ?

**Comment by puzpuzpuz** [comment]
> could we have CI setup ? Would you like help with that ?

Not sure if I follow. CI is there and it's configured via GHA. See https://github.com/questdb/go-questdb-client/runs/7130372131?check_suite_focus=true to find the last build.

File: sender.go (hunk 1 above)
Comment (puzpuzpuz) [reply sender.go:51]: My assumption was that 99.(9)% of users won't change the server-side default. But let me make it configurable since it won't hurt.

**Comment by amunra** [comment]
```java
    public static boolean isValidColumnName(CharSequence seq, int fsFileNameLimit) {
        int l = seq.length();
        if (l > fsFileNameLimit) {
            // Most file systems don't support files name longer than 255 bytes
            return false;
        }
```

I haven't spotted validation of column name length. Have I missed it?

**Comment by amunra** [comment]
> ```java
>     public static boolean isValidColumnName(CharSequence seq, int fsFileNameLimit) {
>         int l = seq.length();
>         if (l > fsFileNameLimit) {
>             // Most file systems don't support files name longer than 255 bytes
>             return false;
>         }
> ```
> 
> I haven't spotted validation of column name length. Have I missed it?

It's in `writeColumnName`, I missed it :-)

**Comment by puzpuzpuz** [comment]
> It's in `writeColumnName`, I missed it :-)

Yup, it's there.

**Review by jerrinot — APPROVED** [review]

**Pull request merged**